hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/snapshot/ExportSnapshot.java

```diff
@@ -996,6 +1000,12 @@ public int doWork() throws IOException {
     LOG.debug("outputFs={}, outputRoot={}, skipTmp={}, initialOutputSnapshotDir={}",
       outputFs, outputRoot.toString(), skipTmp, initialOutputSnapshotDir);
 
+    // Verify snapshot source before copying files
+    if (verifySource) {
+      LOG.info("Verify snapshot source.");
```

Comment:
How about adding `srcConf`, `inputFs`, `inputRoot`, `snapshotDir` values in the log as well?

Reply:
I added inputFs, inputRoot, snapshotDir values in the log, srcConf contains too many things that I didn't add.